src/libs/vue-h-zoom.js

```diff
@@ -1,3 +1,11 @@
+const DEFAULT_BACKGROUND_OPTIONS = {
+  image: 'none',
+  color: '#fff',
+  repeat: false,
+  contain: true,
```

Comment:
I think this flag can be inferred from truthy value of backgroundOptions value, if its truthy, then it will be true, false otherwise

Reply:
sure pak, will update this for `main image` behavior:
truthy value of backgroundOptions will automatically make the `main image` as `contain`. 
however, should we use this option for the ` custom background` itself? suppose that user want to make the `custom background` as cover